platform-hub-api/spec/controllers/projects_controller_spec.rb

```diff
@@ -491,6 +491,93 @@
     end
   end
 
+  describe 'GET #role_check' do
+    let :role do
+      'manager'
```

Comment:
worth adding extra spec for case where passed role is not `manager`?

Reply:
Good idea, though that should be covered by the routing spec (that makes sure it doesn't even make it to the controller if an incorrect role is passed in).